axelor-account/src/main/resources/views/InvoiceLine.xml

```diff
@@ -388,6 +388,8 @@
   </action-group>
 
   <action-group name="action-invoice-line-group-taxline-onchange">
+    <action name="action-invoice-method-check-invoice-lines-not-only-non-deductible-taxes"/>
+    <action name="action-invoice-line-method-check-sum-of-non-deductible-taxes"/>
     <action name="action-invoice-line-method-recompute-tax"/>
     <action name="action-invoice-line-method-convert-tax"/>
     <action name="action-invoice-line-method-compute"/>
```

Comment:
check should not be done in view, remove these actions and add the checks directly in the right service.